assessments/FastBridge_English/earthmover.yaml

```diff
@@ -0,0 +1,2015 @@
+      - ScreeningPeriod5CBMR-EnglishGrowthScoreFromScreeningPeriod4toScreeningPeriod5
+      - ScreeningPeriod5CBMR-EnglishSchoolGrowthPercentileFromScreeningPeriod4toScreeningPeriod5
+      - ScreeningPeriod5CBMR-EnglishDistrictGrowthPercentileFromScreeningPeriod4toScreeningPeriod5
+      - ScreeningPeriod5CBMR-EnglishNationalGrowthPercentileFromScreeningPeriod4toScreeningPeriod5
+      - ScreeningPeriod5CBMR-EnglishGrowthPercentilebyStartScoreFromScreeningPeriod4toScreeningPeriod5
+
+  assessmentReportingMethodDescriptors:
+    file: ./seeds/assessmentReportingMethodDescriptors.csv
+    header_rows: 1
+  assessments:
+    file: ./seeds/assessments.csv
+    header_rows: 1
+  assessmentPeriodDescriptors:
+    file: ./seeds/assessmentPeriodDescriptors.csv
+    header_rows: 1
+  objectiveAssessments:
+    file: ./seeds/objectiveAssessments.csv
+    header_rows: 1
+  performanceLevelDescriptors:
+    file: ./seeds/performanceLevelDescriptors.csv
+    header_rows: 1
+
+transformations:
+  input:
+      source: $sources.input
+      operations: []
+
+  # modify input for basic mappings
+  cleaned_input:
+    source: $transformations.input
+    operations:
+      - operation: add_columns
+        columns:
+          join_key: 1
+          descriptor_namespace_override: ${DESCRIPTOR_NAMESPACE_OVERRIDE}
+      - operation: duplicate_columns
+        columns:
+          AssessmentLanguage: AssessmentLanguageDescriptor
+      # map assessment language to its descriptor
+      - operation: map_values
+        column: AssessmentLanguageDescriptor
+        map_file: ./seeds/mapping_assessment_language.csv
+      # map grades
+      - operation: map_values
+        column: Grade
+        map_file: ./seeds/mapping_grade.csv
+      # convert date values
+      - operation: date_format
+        columns: 
+          - DOB
+        {% for period in assessment_periods %}
+          {% for objective_name in objective_names %}
+          - {{period}}{{objective_name}}FinalDate
+          {% endfor %}
+        {% endfor %}
+        from_format: "%m/%d/%Y"
+        to_format: "%Y-%m-%d"
+      # rename student id column after x-walk operation
+      - operation: rename_columns
+        columns:
+          ${STUDENT_ID_NAME}: student_unique_id
+
+  # >>>>> Modify seed files for namespace override >>>>>
+
+  transformations_seed_file_assessmentPeriodDescriptors:
+    source: $sources.assessmentPeriodDescriptors
+    operations:
+      - operation: modify_columns
+        columns:
+          namespace: "{%raw%}{{namespace | replace('<NAMESPACE_PREFIX>', '${DESCRIPTOR_NAMESPACE_OVERRIDE}')-}}{%endraw%}"
+        
+  transformations_seed_file_assessmentReportingMethodDescriptors:
+    source: $sources.assessmentReportingMethodDescriptors
+    operations:
+      - operation: modify_columns
+        columns:
+          namespace: "{%raw%}{{namespace | replace('<NAMESPACE_PREFIX>', '${DESCRIPTOR_NAMESPACE_OVERRIDE}')-}}{%endraw%}"

```

Comment:
Okay, I see how this is working re: my other comment. For descriptors that are assessment-specific (reporting methods, performance levels, etc.) the namespace should refer to the assessment vendor (in this case Illuminate). those _can_ be hardcoded in the seed, as they are specific to this assessment and won't be overwritten

Reply:
The four descriptors for which you have destinations (category, period, reporting methods, perf levels) will all use illuminate namespace and can be hardcoded in the seeds. All others should use [edfi default values/namespaces](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-Data-Standard/tree/main/Descriptors)